src/index.html

```diff
@@ -1,13 +1,46 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body>
+    <div class="card" data-qa="card">
+      <div class="card__header">
+        <div class="card__content" >
+          <img
+            src="./images/imac.jpeg"
+            alt="imac"
+            class="card__page "
+          >
+          <div class="card__information">
+            <p class="card__text">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</p>
+            <span class="card__product-code" >Product code: 195434</span>
+            <div class="card__container">
+              <div class="card__stars">
+                <div class="card__star-1"></div>
+                <div class="card__star-2"></div>
+                <div class="card__star-3"></div>
+                <div class="card__star-4"></div>
+                <div class="card__star-5"></div>
+                <div class="card__reviews">Reviews: 5</div>
+              </div>
+            </div>
+            <div class="card__cost-content">Price:
+              <div class="card__cost">$2,199</div>
+            </div>
```

Comment:
Use semantic tags for text content like `<p>` for `Price` and `2,199` and put them inside div container